cabal.project

```diff
@@ -8,7 +8,7 @@ packages:
          ./hls-plugin-api
          ./hls-test-utils
 
-index-state: 2024-02-25T00:00:00Z
+index-state: 2024-03-09T08:17:00Z
```

Comment:
Bumping index state just to avoid this warning:
```
Warning: Requested index-state 2024-02-25T00:00:00Z is newer than 
'hackage.haskell.org'! Falling back to older state (2024-02-24T17:42:44Z).
```